sunpy/map/__init__.py

```diff
@@ -8,3 +11,38 @@
 
 from .compositemap import CompositeMap
 from .mapsequence import MapSequence
+
+
+# If the documentation is being built using Sphinx, append output HTML to the docstsrings for the
+#   GenericMap and MapSequence `quicklook()` methods.
+# This code lives here instead of the respective source files because the source files need to be
+#   fully imported before the following code will successfully run.
+# This code detects a Sphinx build via the simplistic approach of checking if Sphinx has been
+#   imported, and thus there is the potential for false positives.
+if 'sphinx' in sys.modules:
```

Comment:
This is polluting this namespace a fair bit, might be nice if it were just a private function call.

Reply:
Thinking about this some more, technically this approach doesn't actually pollute the namespace at all.  A user who doesn't import Sphinx won't add any of these new variables and nothing will show up through tab completion.  A Sphinx build does create these variables, but they don't get picked up by our docs, so that pollution is entirely hidden.

Defining a function, even a private one, would pollute the namespace more than what's currently done.  Thoughts?

Reply:
After considering the benefits for future maintenance, I have moved the code to private class methods.